QueryBuilder/Query.Aggregate.cs

```diff
@@ -82,6 +82,17 @@ public Query SelectCountDistinct(IEnumerable<string> columns, string alias = nul
         }
 
 
+        /**********************************************************************
+         ** AnyValue                                                         **
+         **********************************************************************/
+        // Note: The DISTINCT keyword can be specified for this function, but it
+        // does not have any effect.
+        public Query SelectAnyValue(string column, string alias = null)
+        {
+            return SelectAggregate("anyvalue", new[] { column }, AggregateColumn.AggregateDistinct.aggregateNonDistinct, alias);
```

Comment:
Any reason not to use `"any_value"` here? Feels i bit weird to not match the sql here.

Reply:
rebased on top of #10, now always uses `any_value`, but the type is also 'encoded' in the actual type and not actually used anywhere directly, so it is somewhat redundant.